### DISCUSSION TITLE
Helfer Info Fenster

### DISCUSSION THREAD

**Discussion by MacRama1** [Courseplay/Courseplay_FS22#1219] [General]
Hallo CP Team,

vielen lieben Dank dass, das Info Fenstern endlich wieder da ist, das hat wirklich gefehlt.
Leider kann man auch die Fahrzeuge anderer Mitspieler auf dem Server sehen. 
Es gibt auch Mitspieler die sich wirklich Zeit lassen bis sie sich Ihrer Fahrzeuge widmen (bei meinem Mitspieler schon seit einer Stunde der Fall) und man hat dann pausenlos die Info deren Fahrzeuge dort stehen.

Lässt sich das ändern so das man immer nur die Info seiner eigenen Fahrzeuge bekommt?

Gruß Mac

**Comment by Tensuko** [comment]
Glaube schon das es geht, über die FarmID.
@schwiti6190

**Reply by MacRama1** [reply]
Ergänzung:

Habe meinen Mitspieler eben darum gebeten mal nach seinem Traktor zu sehen und er sagte das dieser Traktor längst eine andere Aufgabe erledigt.
Bei mir steht aber immer noch ein und dieselbe Info. (Arbeit erledigt).
Und jetzt ist auch noch mein Traktor dazu gekommen (der mittlerweile auf einem anderen Feld arbeitet).
Habe also jetzt dauerhaft die Info beider Traktoren.

Soll ich ein Issue draus machen??

Screenshot im Anhang
![fsScreen_2022_03_16_16_00_49](https://user-images.githubusercontent.com/99003248/158620805-62dc53ca-bdce-47c8-8c95-702e2b08fc67.png)

**Reply by Tensuko** [reply]
> Soll ich ein Issue draus machen??

Schon erledigt.
